src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int numRandCol = Color.values().length;
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(numRandCol);
+        return String.valueOf(Color.values()[index]);
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants